src/Interpreters/Session.h

```diff
@@ -90,6 +93,7 @@ class Session
 
     mutable UserPtr user;
     std::optional<UUID> user_id;
+    std::vector<UUID> extra_granted_roles;
```

Comment:
If I understood correctly, `additional roles` == `extra_roles` == `extra_granted_roles`? Is it correct?

Reply:
Agree, renaming all that to just `external_roles`